### DISCUSSION TITLE
System requirments

### DISCUSSION THREAD

**Discussion by ebenfarnworth** [coqui-ai/TTS#1116] [General Q&A]
What are the recommended system requirements for using coqui as a single user on a desktop computer? (not voice training, simply using it as a TTS engine to read text on the fly).
I have created this [tutorial](https://ebenfarnworth.substack.com/p/text-to-speech-tts-on-linux) for other (non machine learning TTS programs) on KDE and gnome integration and I was hoping to make a similar setup using coqui. I was able to install coqui on my system and downloaded some voices, but it took over a minute on my old i3 system with 4gb of ram to get the audio output from a short paragraph of text. What sort of system would be required to run coqui and get near real time speech output? many thanks.

**Comment by erogol** [comment]
I don't have specs but you can run it from raspi to cluster size machines. Windows is a bit problematic.

**Reply by ebenfarnworth** [reply]
Thanks for writing. I get that it can work on many different size machines. My question was what hardware is required for near real time tts?
As I said, I have it running on an old I3 with 4gb of RAM. This question is not related to Windows, I'm currently running it on Arch with KDE.
